.github/workflows/ci.yml

```diff
@@ -13,27 +13,15 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install Nix
         uses: DeterminateSystems/nix-installer-action@main
-      - name: Record existing bundle hash
-        run: |
-          echo "BUNDLE_HASH=$(sha256sum <dist/index.js | sed 's/  -//')" >>$GITHUB_ENV
+      - uses: DeterminateSystems/magic-nix-cache-action@main
       - name: Check shell scripts
```

Comment:
Can we have this more fine grained, like here?

https://github.com/DeterminateSystems/detsys-ts/blob/main/.github/workflows/ci.yml